clr-k8s-examples/10-cri-resource-manager/README.md

```diff
@@ -0,0 +1,55 @@
+CRI Resource Manager
+====================
```

Comment:
Oh, apparently you are using markdown [setext](https://github.github.com/gfm/#setext-heading) heading markers (`=====` and `-------` for instance), and not [atx](https://github.github.com/gfm/#atx-heading) heading markers (`#`, `##` etc.) - which is what I think we use for all other docs in this repo. I didn't even know you could use `setext` in markdown :-)

Reply:
done.